src/cluster.js

```diff
@@ -52,7 +52,7 @@ function Cluster(instance, name) {
  *
  * @private
  *
- * @param {string} project The project.
+ * @param {string} project The project ID.
```

Comment:
This is right.  Please also add a check in the code to make sure that there isn't a '/'.  If there is, please do a slice and get the last component.  That should be done for both location and project.